inc/authldap.class.php

```diff
@@ -1696,9 +1851,13 @@ static function getAllUsers(array $options, &$results, &$limitexceeded) {
          foreach ($diff as $user) {
             //If user dn exists in DB, it means that user login field has changed
             if (!$tmpuser->getFromDBByDn(toolbox::addslashes_deep($user_infos[$user]["user_dn"]))) {
-               $list[] = ["user"      => $user,
-                               "timestamp" => $user_infos[$user]["timestamp"],
-                               "date_sync" => Dropdown::EMPTY_VALUE];
+               $entry  = ["user"      => $user_infos[$user][$config_ldap->fields['login_field']],
+                          "timestamp" => $user_infos[$user]["timestamp"],
+                          "date_sync" => Dropdown::EMPTY_VALUE];
+               if ($config_ldap->isSyncFieldEnabled()) {
+                  $entry[$config_ldap->getLdapIdentifierToUse()] = $user_infos[$user][$config_ldap->fields['sync_field']];
```

Comment:
``$config_ldap->getLdapIdentifierToUse()`` -> ``$field_for_sync``
same with ``$config_ldap->fields['sync_field']``.

So, maybe i'm wrong but the line could be reduced to 
```
 $entry[$field_for_sync] = $user_infos[$user][$field_for_sync];
```